akeneo-design-system/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,127 @@
+import React, {Ref} from 'react';
+import styled from 'styled-components';
+import {AkeneoThemedProps, getColor, getFontSize} from '../../theme';
+
+const ProgressBarContainer = styled.div``;
+
+const Header = styled.div`
+  display: flex;
+  align-items: stretch;
+  flex-direction: row;
+  font-size: ${getFontSize('default')};
+`;
+
+const Title = styled.div`
+  color: ${getColor('grey140')};
+  flex-shrink: 1;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  flex-grow: 1;
+  padding-right: 20px;
+`;
+
+const ProgressLabel = styled.div`
+  color: ${getColor('grey120')};
+  flex-shrink: 0;
+`;
+
+const ProgressBarBackground = styled.div<{height: ProgressBarHeight} & AkeneoThemedProps>`
+  background: ${getColor('grey80')};
+  height: ${props => getHeight(props.height)};
+  overflow: hidden;
+  position: relative;
+`;
+
+const ProgressBarFill = styled.div<{width: number; color: string}>`
+  background: ${props => props.color};
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: width 0.3s;
+  width: ${props => props.width.toString() + '%'};
+`;
+
+const getHeight = (height: ProgressBarHeight): string => {
+  switch (height) {
+    case 'large':
+      return '10px';
+    case 'small':
+    default:
+      return '4px';
+  }
+};
+
+const sanitizePercent = (percent: number): number => {
+  if (percent < 0) {
+    return 0;
+  }
+
+  if (percent > 100) {
+    return 100;
+  }
+
+  return percent;
+};
+
+type ProgressBarHeight = 'small' | 'large';
+
+type ProgressBarProps = {
+  /**
+   * Define the color of the progress bar.
+   */
+  color: string;
+
+  /**
+   * The progression of the progress bar in percentage (from 0 to 100).
+   */
+  percent: number;
+
+  /**
+   * The progress bar title.
+   */
+  title?: string;
+
+  /**
+   * Describe the progress with a label (example: 46%, 30 minutes left).
+   */
+  progressLabel?: string;
+
+  /**
+   * Define the height of the progress bar.
+   */
+  height?: ProgressBarHeight;
```

Comment:
```suggestion
  size?: ProgressBarHeight;
```
To be consistent with other components (Button)?

Reply:
The difference with button is that `size` is really about its size, both height & width. Since the progress bar always has 100% width, the variation is on the height. I though I would be better to be explicit.

Reply:
I think we should use size as it's a finished value (small or large) and not a value in pixel. I aslo think that we should be consistent with button. When I say that I want a small progress bar, I understand that it's only about the rendering and not the width of the progress bar